README.md

```diff
@@ -15,6 +15,14 @@
 # Nexus Repository Manger p2 Format
 [![Maven Central](https://img.shields.io/maven-central/v/org.sonatype.nexus.plugins/nexus-repository-p2.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22org.sonatype.nexus.plugins%22%20AND%20a:%22nexus-repository-p2%22) [![CircleCI](https://circleci.com/gh/sonatype-nexus-community/nexus-repository-p2.svg?style=shield)](https://circleci.com/gh/sonatype-nexus-community/nexus-repository-p2) [![Join the chat at https://gitter.im/sonatype/nexus-developers](https://badges.gitter.im/sonatype/nexus-developers.svg)](https://gitter.im/sonatype/nexus-developers?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![DepShield Badge](https://depshield.sonatype.org/badges/sonatype-nexus-community/nexus-repository-p2/depshield.svg)](https://depshield.github.io)
 
+> **Huzzah!** p2 is now part of Nexus Repository Manager. Version 3.20 includes the p2 plugin by default.
```

Comment:
Unless you're recording something else, I think p2 was 3.21 (ref https://help.sonatype.com/display/NXRM3/Release+Notes#ReleaseNotes-NexusRepositoryManager3.21.0)

Reply:
Updated https://github.com/sonatype-nexus-community/nexus-repository-p2/pull/79/commits/badc6e7b769fdef8cc224cca99df41dcd02b6390